metricbeat/module/mongodb/collstats/collstats.go

```diff
@@ -32,74 +35,84 @@ func init() {
 	)
 }
 
-// MetricSet type defines all fields of the MetricSet
+// Metricset type defines all fields of the Metricset
 // As a minimum it must inherit the mb.BaseMetricSet fields, but can be extended with
 // additional entries. These variables can be used to persist data or configuration between
 // multiple fetch calls.
-type MetricSet struct {
-	*mongodb.MetricSet
+type Metricset struct {
```

Comment:
Is there a reason for this name change? There appears to be disagreement in the code base about how this should be spelled; the docs spell it as a single word, but the mb package spells it as a camel-cased pair.